graylog2-web-interface/src/components/indices/IndexSetDetails.tsx

```diff
@@ -21,35 +21,55 @@ import HideOnCloud from 'util/conditional/HideOnCloud';
 import { Col } from 'components/bootstrap';
 import { IndicesConfiguration } from 'components/indices';
 import type { IndexSet } from 'stores/indices/IndexSetsStore';
+import useProfile from 'components/indices/IndexSetFieldTypeProfiles/hooks/useProfile';
+import { Link } from 'components/common/router';
+import Routes from 'routing/Routes';
 
 import StyledIndexSetDetailsRow from './StyledIndexSetDetailsRow';
 
 type Props = {
   indexSet: IndexSet,
 };
 
-const IndexSetDetails = ({ indexSet }: Props) => (
-  <StyledIndexSetDetailsRow>
-    <Col lg={3}>
-      <dl>
-        <dt>Index prefix:</dt>
-        <dd>{indexSet.index_prefix}</dd>
-        <HideOnCloud>
-          <dt>Shards:</dt>
-          <dd>{indexSet.shards}</dd>
-          <dt>Replicas:</dt>
-          <dd>{indexSet.replicas}</dd>
-        </HideOnCloud>
-        <dt>Field type refresh interval:</dt>
-        <dd>{indexSet.field_type_refresh_interval / 1000.0} seconds</dd>
-      </dl>
-    </Col>
+const IndexSetDetails = ({ indexSet }: Props) => {
+  const { data: { name: profileName }, isFetched } = useProfile(indexSet.field_type_profile);
 
-    <Col lg={6}>
-      <IndicesConfiguration indexSet={indexSet} />
-    </Col>
-  </StyledIndexSetDetailsRow>
-);
+  return (
+    <StyledIndexSetDetailsRow>
+      <Col lg={3}>
+        <dl>
+          <dt>Index prefix:</dt>
+          <dd>{indexSet.index_prefix}</dd>
+          <HideOnCloud>
+            <dt>Shards:</dt>
+            <dd>{indexSet.shards}</dd>
+            <dt>Replicas:</dt>
+            <dd>{indexSet.replicas}</dd>
+          </HideOnCloud>
+          <dt>Field type refresh interval:</dt>
+          <dd>{indexSet.field_type_refresh_interval / 1000.0} seconds</dd>
+          <dt>Field type profile:</dt>
+          {isFetched && (
+          <dd>
+            {indexSet.field_type_profile
+              ? (
+                <Link to={Routes.SYSTEM.INDICES.FIELD_TYPE_PROFILES.edit(indexSet.field_type_profile)}
+                      target="_blank">
+                  {profileName}
+                </Link>
+              )
+              : 'Not set'}
```

Comment:
In my case "Not set" is not being displayed
![image](https://github.com/Graylog2/graylog2-server/assets/46300478/4faff068-7863-4046-98f2-d53b8da361ff)


Maybe let's display it italic as well, like on the index set field types management page:
![image](https://github.com/Graylog2/graylog2-server/assets/46300478/d655bac0-801b-473a-ad80-2b70a7e2a29e)